cmd/errtrace/main_test.go

```diff
@@ -780,14 +779,43 @@ func TestStdinNoInputMessage(t *testing.T) {
 			if want := 0; exitCode != want {
 				t.Errorf("exit code = %d, want %d", exitCode, want)
 			}
+			<-done
+
+			// errtrace's goroutine may still be writing to stderr.
+			// Try a few times before giving up.
```

Comment:
I don't think there's an easy fix, but I'm really starting to think the timeout appraoch to the stdin warn isn't worth the complexity -- the read code has more noise, and now we have to deal with a possible background goroutine still writing to stderr even though teh `mainCmd.Run` returned.

I think we should land this as-is to get it fixed, but consider other alternatives that avoid background goroutines after main returns.